bgpd/bgp_routemap.c

```diff
@@ -1148,7 +1148,7 @@ route_match_vrl_source_vrf(void *rule, const struct prefix *prefix,
 	if (strncmp(vrf_name, "n/a", VRF_NAMSIZ) == 0)
 		return RMAP_NOMATCH;
 
-	if (path->extra == NULL)
+	if (path->extra == NULL || path->extra->bgp_orig == NULL)
 		return RMAP_NOMATCH;
```

Comment:
this should be separate from rest of the code
it looks like a sanity code ?

Reply:
EVPN routes marked as imported routes, having bgp path info's extra,
where as they are not truly imported routes, so original bgp info will be NULL.

Thats the reason added the Null check for the original bgp info(bgp_path_info_nexthop_cmp bgp_mpath.c:L189).
Also added the same check here for route-maps.

We can always segregate this if required.

Reply:
yes, please. add a specific commit for that, please